dev-docs/source/GitConfig.rst

```diff
@@ -0,0 +1,168 @@
+.. _GitConfig:
+
+=================
+Mantid Git Config
+=================
+
+.. contents:: Contents
+   :local:
+
+Summary
+-------
+
+This page describes some configuration that will help with your usage of Git - some of which is strongly recommended. This should all be added to the ``.gitconfig`` file in your user home directory, i.e. ``~/`` on Linux or ``%USERPROFILE%`` on Windows. This will then be global across all of your repositories - please sanity check these suggestions if you use git for other repositories.
+
+Note that some of the values are commented out because they must be updated with your own details or are OS specific.
+
+Recommended Config
+------------------
+
+It is recommended to set up the following as a minimum:
+
+.. code-block:: sh
+
+   [user]
+           # Uncomment and set these to your own details
+           #name =
+           #email =
+   [core]
+           # Set an editor you like
+           editor = vi
```

Comment:
IMO if a user is unfamiliar with git configs I would personally push them towards `nano` on Linux and `notepad++` on Windows as their default editors